src/App.js

```diff
@@ -21,6 +22,7 @@ function App() {
         <Routes>
           <Route path='/' element={<Articles />} />
           <Route path='/:topic' element={<ArticleByCatergory />} />
+          <Route path='/article/:article_id' element={<ArticleById />} />
```

Comment:
its really picky but articles/:article_id makes a little more sense to me if I were thinking about this like a folder structure